addons/udes_get_info/models/models.py

```diff
@@ -55,7 +55,7 @@ def _get_info(self, level=1, max_level=None, info_fields=None, extra_fields=None
             if level == max_level:
                 next_info_fields = {"info_fields": BASIC_GET_INFO_VALUES}
             else:
-                next_info_fields = {"extra_fields": extra_fields}
+                next_info_fields = {"info_fields": info_fields}
```

Comment:
If you pass info_fields on relational field i think that on next iteration it wont pick the info_fields of the relational field model

Reply:
Possibly :)  I'm going to revert this change for now and rework the tests so that they pass instead; I'll log an issue to get this fixed properly.

Reply:
Workaround added in [d660cc5](https://github.com/unipartdigital/udes-open/pull/604/commits/d660cc52d5c9122ee2aaec240e3675054e216a10)